docs/modules/airflow/examples/example-airflow-kubernetes-executor-s3-logging.yaml

```
@@ -0,0 +1,24 @@
+apiVersion: airflow.stackable.tech/v1alpha1
+kind: AirflowCluster
+metadata:
+  name: airflow
+spec:
+  image:
+    productVersion: 2.9.3
+  clusterConfig: {}
+  webservers:
+    roleGroups:
+      default:
+        envOverrides: &s3-logging-env-overrides
+          AIRFLOW_LOGGING_REMOTE_LOGGING: "True"
+          AIRFLOW_LOGGING_REMOTE_BASE_LOG_FOLDER: s3://<bucket-name>/airflow-task-logs/
+          # The name / connection ID created in the Airflow Web UI
+          AIRFLOW_LOGGING_REMOTE_LOG_CONN_ID: minio
+        replicas: 1
+  schedulers:
+    roleGroups:
+      default:
+        envOverrides: *s3-logging-env-overrides
+        replicas: 1
```

Sorry for the nitpick again!
I personally prefer to configure everything on the role (as long as it#s not rolegroup specific)

This is optional, you don't need to apply this!

```suggestion
  webservers:
    envOverrides: &s3-logging-env-overrides
      AIRFLOW_LOGGING_REMOTE_LOGGING: "True"
      AIRFLOW_LOGGING_REMOTE_BASE_LOG_FOLDER: s3://<bucket-name>/airflow-task-logs/
      # The name / connection ID created in the Airflow Web UI
      AIRFLOW_LOGGING_REMOTE_LOG_CONN_ID: minio
    roleGroups:
      default:
        replicas: 1
  schedulers:
    envOverrides: *s3-logging-env-overrides
    roleGroups:
      default:
        replicas: 1
```

---

Well actually makes it more readable, so ill take it :)